### ISSUE TITLE
Page Intro Copy Update

### ISSUE THREAD

**Issue by reconwhispers** [KidSysco/dig-a-hash-2#7]
Update COPY

Business Infrastructure for Web3

Our Mission is to assist small businesses, owner operators, creators, and entrepreneurs in migrating their products and services into the emerging web3 landscape. 

We look to accomplish this for our Clients through 3 core service initiatives:

1-Onboarding education to explain:
- What NFTs, Web3, Metaverse, Augmented Reality, DAOs, Defi, and Blockchain are
- How they are currently being used
- What it means for the future of your business
- What you need to get started

2-Strategy and business development with the focus on emerging into web3 for long term growth and stability

3-Execution through tech leveraged solutions 

![image](https://user-images.githubusercontent.com/97454558/180659034-ba9e7051-f4c0-44b2-b8ed-8debcddab0f3.png)

**Comment by KidSysco** [comment]
I felt this was too long so I only took the mission statement. However, the rest of this is really good, I just think it belongs somewhere else. We can work on that next.

**Comment by reconwhispers** [comment]
Business Infrastructure for Web3

Our Mission is to assist businesses, owner/operators, creators, and entrepreneurs in migrating their products and services into the emerging web3 landscape. 

3 core service initiatives:

1-Onboarding Education

2-Strategy and Business Development with the focus on emerging into web3 for long term growth and stability

3-Execution through Tech Leveraged Solutions

**Comment by reconwhispers** [comment]
still too long?

**Comment by KidSysco** [comment]
Yes, but I love it. Most sites will have a section on the navbar called "Services".

The Services page details the offerings, options, pricing, etc.

Each service gets it's own page and navbar link.

But the front page tagline/mission statement should really be one sentence if possible. 25 words or less is real good. I think it is perfect. Now let's design and talk about services we offer.
